pubspec.yaml

```diff
@@ -29,13 +29,13 @@ dependencies:
   flutter_bloc: ^1.0.0
   json_annotation: ^3.0.0
   dio: ^3.0.4
-
+  font_awesome_flutter :
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
   json_serializable: ^3.2.3
-  build_runner: ^1.7.2
+  build_runner: ^1.7.1  #(Edit by Maadhav due to error with 1.7.2 version)
```

Comment:
Don't write explanations comments in files, if something has to be explained write it with pull request